packages/dds/matrix/src/matrix.ts

```diff
@@ -323,9 +323,12 @@ export class SharedMatrix<T = any>
 		//       yields an ~40% improvement in the case of a cache hit (node v12 x64)
 
 		// Map the logical (row, col) to associated storage handles.
-		const rowHandle = this.rowHandles.getHandle(row);
+		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
You shouldn't assert in these locations - instead, `getHandle`'s return type should be explicitly made `Handle`, as it's not expected to return `undefined`.  And then you'll need a non-null assertion inside of `getHandle` probably.